terms-of-service.md

```diff
@@ -0,0 +1,213 @@
+
+    5. your Contributions are not unsolicited or unauthorized advertising, promotional materials, pyramid schemes, chain letters, spam, mass mailings, or other forms of solicitation.
+
+    6. your Contributions are not obscene, lewd, lascivious, filthy, violent, harassing, libelous, slanderous, or otherwise objectionable (as determined by us).
+
+    7. your Contributions do not ridicule, mock, disparage, intimidate, or abuse anyone.
+
+    8. your Contributions do not advocate the violent overthrow of any government or incite, encourage, or threaten physical harm against another.
+
+    9. your Contributions do not violate any applicable law, regulation, or rule.
+
+    10. your Contributions do not violate the privacy or publicity rights of any third party.
+
+    11. your Contributions do not contain any material that solicits personal information from anyone under the age of 18 or exploits people under the age of 18 in a sexual or violent manner.
+
+    12. your Contributions do not violate any federal or state law concerning child pornography, or otherwise intended to protect the health or well-being of minors.
+    
+    13. your Contributions do not include any offensive comments that are connected to race, national origin, gender, sexual preference, or physical handicap.
+
+    14. your Contributions do not otherwise violate, or link to material that violates, any provision of these Terms of Service, or any applicable law or regulation.
+
+Any use of the Site in violation of the foregoing violates these Terms of Service and may result in, among other things, termination or suspension of your rights to use the Site. 
+ 
+## CONTRIBUTION LICENSE
+
+By posting your Contributions to any part of the Site, you automatically grant, and you represent and warrant that you have the right to grant, to us an unrestricted, unlimited, irrevocable, perpetual, non-exclusive, transferable, royalty-free, fully-paid, worldwide right, and license to host, use, copy, reproduce, disclose, resell, publish, broadcast, retitle, archive, store, cache, publicly perform, publicly display, reformat, translate, transmit, excerpt (in whole or in part), and distribute such Contributions (including, without limitation, your image and voice) for any purpose, commercial, advertising, or otherwise, and to prepare derivative works of, or incorporate into other works, such Contributions, and grant and authorize sublicenses of the foregoing. The use and distribution may occur in any media formats and through any media channels. 
+
+This license will apply to any form, media, or technology now known or hereafter developed, and includes our use of your name, company name, and franchise name, as applicable, and any of the trademarks, service marks, trade names, logos, and personal and commercial images you provide. You waive all moral rights in your Contributions, and you warrant that moral rights have not otherwise been asserted in your Contributions. 
```

Comment:
We should remove "resell", "commercial", "advertising" and anything else that's related to monetizing contributions for our monetary gains.